src/Conversion/ONNXToKrnl/Math/Softmax.cpp

```diff
@@ -126,6 +126,7 @@ static void emitInstForSoftmaxBeforeV13(ConversionPatternRewriter &rewriter,
   KrnlBuilder createKrnl(rewriter, loc);
   IndexExprScope ieScope(createKrnl);
   MemRefBoundsIndexCapture inputBounds(input);
+  LiteralIndexExpr zeroIE(0);
```

Comment:
[nit]: some files in this PR use the name `zero` and others `zeroIE` for the same expr. Optional: make them consistent.

Reply:
The reason I use `zeroIE` instead of `zero` is `zero` is already used. I can change all `zero` index expr to `zeroIE` instead.

Reply:
ok got it, no don't bother.

Reply:
Changed it anyways.